bwrap.xml

```diff
@@ -190,10 +190,21 @@
   <para>
     Filesystem related options. These are all operations that modify the filesystem directly, or
     mounts stuff in the filesystem. These are applied in the order they are given as arguments.
+  </para>
+  <para>
     Any missing parent directories that are required to create a specified destination are
-    automatically created as needed.
+    automatically created as needed. Their permissions are normally set to 0755
+    (rwxr-xr-x). However, if a <option>--perms</option> option is in effect, and
+    it sets the permissions for group or other to zero, then parent directories
+    will also have their permissions set to zero.
   </para>
   <variablelist>
+    <varlistentry>
+      <term><option>--perms <arg choice="plain">OCTAL</arg></option></term>
+      <listitem><para>This option does nothing on its own, and must be followed
+        by one of the options that it affects. It sets the permissions
+      of that file to <arg choice="plain">OCTAL</arg>.</para></listitem>
```

Comment:
```suggestion
        of that file to <arg choice="plain">OCTAL</arg>.</para></listitem>
```

Perhaps I should say explicitly here that `--perms` only affects the *next* operation, and then is forgotten.